### DISCUSSION TITLE
Allow For Multiple Answers To Be Marked as Answer

### DISCUSSION THREAD

**Discussion by GrahamSH-LLK** [community/community#9] [Discussions]
In discussions, sometimes there isn't only one good answer. In that case, you should be able to mark multiple answers as correct. Sadly, you can't. Please add this functionality.

**Comment by kilitary** [comment]
yeah

**Comment by Griffin641498** [comment]
> In discussions, sometimes there isn't only one good answer. In that case, you should be able to mark multiple answers as correct. Sadly, you can't. Please add this functionality.

**Comment by tvews8z** [comment]
> In discussions, sometimes there isn't only one good answer. In that case, you should be able to mark multiple answers as correct. Sadly, you can't. Please add this functionality.

**Comment by MrSarno** [comment]
Strongly support this suggestion. 👍

**Comment by lwi666** [comment]
I haven't get verification email . I noticed when login azure with github

**Comment by lwi666** [comment]
Showing email not verified request again

**Comment by kilitary** [comment]
some on e can suggest a n-solution
